File: 240530/함수를 이용한 소수 판별/decimal-decisions-using-functions.py
def is_check(num):
    check = True
    for i in range(2, num):
        if num % i == 0:
            check = False
    return check

a, b = tuple(map(int, input().split()))

sum = 0

for num in range(a, b+1):
    if is_check(num):
        sum += num

print(sum)

File: 241230/함수를 이용한 소수 판별/decimal-decisions-using-functions.cpp
#include <iostream>
using namespace std;

bool F(int num) {
    int check = true;
    for (int i=2; i<num; i++) {
        if (num%i == 0) {
            check = false;
            break;
        }
    }
    return check;
}

int main() {
    // 여기에 코드를 작성해주세요.
    int a, b, sum_val=0;
    cin >> a >> b;

    for (; a<=b; a++) {
        if (F(a) && a != 1) sum_val+=a;
    }
    cout << sum_val;
    return 0;
}